pallets/route-executor/src/lib.rs

```diff
@@ -318,6 +351,10 @@ pub mod pallet {
 				)?;
 			}
 
+			if skip_ed_disabling {
+				SkipEd::<T>::kill();
```

Comment:
you should make sure the SkipEd storage is always killed after the last hop is processed, so there is no impact on the on-chain storage